src/github/folderRepositoryManager.ts

```diff
@@ -724,6 +724,10 @@ export class FolderRepositoryManager implements vscode.Disposable {
 		return this._fetchAssignableUsersPromise;
 	}
 
+	async getPullRequestParticipants(githubRepository: GitHubRepository, pullRequestNumber: number): Promise<[IAccount[], IAccount]> {
```

Comment:
I have a slight preference for returning an object like `{suggestions: IAccount[], viewer: IAccount}` as it is possible to see exactly what is being returned by looking at the method signature.